src/Dotnet.Script.Core/Scaffolder.cs

```diff
@@ -100,14 +119,27 @@ private void CreateLaunchConfiguration(string currentWorkingDirectory)
                 Directory.CreateDirectory(vsCodeDirectory);
             }
 
+            // on windows we use this as opportunity to make the file association for .csx -> dotnet-script
+            // (If/when dotnet install command provides us an install time hook this code should move there)
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // check to see if .csx is mapped to dotnet-script
+                if (_commandRunner.Execute("reg", "query HKCU\\Software\\classes\\.csx") != 0)
```

Comment:
I assume we always have the necessary permission to write to the registry?

Reply:
yeah it works under user permissions